packages/dev/core/src/Meshes/Node/Blocks/linearClonerBlock.ts

```diff
@@ -0,0 +1,213 @@
+import { NodeGeometryBlock } from "../nodeGeometryBlock";
+import type { NodeGeometryConnectionPoint } from "../nodeGeometryBlockConnectionPoint";
+import { RegisterClass } from "../../../Misc/typeStore";
+import { NodeGeometryBlockConnectionPointTypes } from "../Enums/nodeGeometryConnectionPointTypes";
+import { Matrix, Quaternion, Vector3 } from "../../../Maths/math.vector";
+import { type NodeGeometryBuildState } from "../nodeGeometryBuildState";
+import type { VertexData } from "core/Meshes/mesh.vertexData";
+import { type Nullable } from "core/types";
+import { PropertyTypeForEdition, editableInPropertyPage } from "core/Decorators/nodeDecorator";
+
+/**
+ * Block used to clone geometry along a line
+ */
+export class LinearClonerBlock extends NodeGeometryBlock {

```

Comment:
you need to implement "implements INodeGeometryExecutionContext, INodeGeometryInstancingContext"
See here: https://github.com/BabylonJS/Babylon.js/blob/58a04329c4d1f591d4c783c694541a7dff0e62e6/packages/dev/core/src/Meshes/Node/Blocks/Instances/instantiateBlock.ts#L15